apps/builder/app/canvas/instance-hovering.ts

```diff
@@ -11,7 +11,11 @@ import type { InstanceSelector } from "~/shared/tree-utils";
 
 type TimeoutId = undefined | ReturnType<typeof setTimeout>;
 
-export const subscribeInstanceHovering = () => {
+export const subscribeInstanceHovering = ({
+  signal,
+}: {
+  signal: AbortSignal;
+}) => {
```

Comment:
I would not mix both abort signal and unsubscribe

Reply:
oversight